.github/workflows/est-ds-realm-test.yml

```diff
@@ -393,17 +393,39 @@ jobs:
         run: |
           docker exec pki mkdir new_certs3
           docker exec -e EST_OPENSSL_CACERT=cacert.pem pki estclient -e -s pki.example.com -p 8443 \
-              --common-name test-new.example.com -o new_certs3 -u est-test-user -h Secret.123
+              --common-name test-new.example.com -o new_certs3 -u est-test-user -h Secret.123 | tee output || true
 
-          docker exec pki openssl base64 -d --in new_certs3/cert-0-0.pkcs7 --out new_certs3/cert-0-0.pkcs7.der
-          docker exec pki openssl pkcs7 -in new_certs3/cert-0-0.pkcs7.der -inform DER -print_certs -quiet -out new_certs3/cert.pem
-          docker exec pki openssl x509 -in new_certs3/cert.pem -subject -noout | tee actual
-          echo "subject=CN=test-new.example.com" > expected
-          diff expected actual
+          # Request should fails with bad request error (status code 400, BAD REQUEST)
+          STATUS=$(cat output  | sed -nE 's/.* HTTP response from EST server was (.*)$/\1/p')
+          [ "$STATUS" == "BAD REQUEST" ]
 
       - name: Enroll new certificate with certificate but using different subject and check disabled
```

Comment:
same question here about "check disabled".

Reply:
As above. I could add something to be more explicit